Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift

```diff
@@ -44,6 +44,24 @@ open class LocationAnnotationNode: LocationNode {
         self.init(location: location, image: view.image)
     }
 
+    public init(location: CLLocation?, layer: CALayer) {
+        let plane = SCNPlane(width: layer.bounds.size.width / 100, height: layer.bounds.size.height / 100)
+        plane.firstMaterial!.diffuse.contents = layer
```

Comment:
Don't use force unwraps, please

Reply:
I actually followed exactly the same as the initialiser for the `UIImage` ;) (L18~)

Yup, definitely I will avoid force unwrap, and I will think about how we can avoid duplicate coding here.

Reply:
Thanks! I haven't had a chance to clean up all the force unwraps yet...  😞

Reply:
you can literally just replace `!` with `?` here